### ISSUE TITLE
Not able to communicate with two devices at once when using node bridge

### ISSUE THREAD

**Issue by bosomt** [trezor/trezor-suite#14285]
**Describe the bug**
Not able to communicate with two devices at once when using node bridge.
You can break communication easily when using two devices at once.
Even reconnect of second device does not help.
Sometimes it works sometimes it dont, so im not able to replicate it everytime

**Steps to reproduce:**
1. Connect two devices /TT and TS5 were used/
2. Start suite with EAP /node bridge enabled/ 
3. Discover both wallets
4. Reload Suite

**Info:**
 - Suite version: desktop 24.9.1 (82c6d9b468f417d99f99d90b3c77d0f02650616b)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/24.9.1 Chrome/124.0.6367.243 Electron/30.3.1 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.2 regular (revision 37006592dbc3f2d817def0c75ed6a62513895ec5)
 - Transport: BridgeTransport 3.0.0

**Screenshots:**
<img width="876" alt="image" src="https://github.com/user-attachments/assets/27d434d7-0480-4369-a73c-c12e13766eb6">

![image](https://github.com/user-attachments/assets/1c141288-81fe-4cde-9e62-b37400a909d2)

![image](https://github.com/user-attachments/assets/44d2216a-f2df-4c7d-ab20-470b47e69af8)

**Comment by mroz22** [comment]
Somewhat simplified steps:

1. 2 devices, connected unlocked
2. start suite-desktop
3. observe that only one device is emitted to suite

so far I was able to observe that this enumeration here
https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/device/DeviceList.ts#L336
correctly returns 2 descriptors
but things start to be weird roughly here 
https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/device/DeviceList.ts#L215 
where `_createAndSaveDevice` is called only once. 

maybe @marekrjpolak could take a quick look please? it could be related to your recent refactorings

**Comment by marekrjpolak** [comment]
![image](https://github.com/user-attachments/assets/e81bbf69-1919-45e5-9ff0-a824ef6fff27)

**Comment by bosomt** [comment]
Managed to replicate it in Brave browser when webUSB is used as transport, 

![Image](https://github.com/user-attachments/assets/472630cb-4d55-4adb-b68a-59a51bc2ebbf)
![Image](https://github.com/user-attachments/assets/071281b2-2c9a-4c73-ab5c-b4531c61f0c6)

**Comment by bosomt** [comment]
After I connected two devices and reloaded Suite app i only see one device in app but two in bridge status prage.
Before app reload i saw and i was able to discover both devices.
After i reconnected second device and i see both of them in app

<img width="974" alt="image" src="https://github.com/user-attachments/assets/1e74a422-b3d3-4caf-b3a2-7d2aedac2239">

**Info:**
 - Suite version: desktop 24.10.0 (7f52887ff9090019573ec3dd3e8a8d210a3a2a00)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/24.10.0 Chrome/124.0.6367.243 Electron/30.3.1 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.3 regular (revision 7f373ae71eca855dd41b4a0fdcc7cadaa13a8281)
 - Transport: BridgeTransport 3.0.0-bundled.24.10.0

**Comment by mroz22** [comment]
Re-opening then. I jus tested:
1. connect 2 devices
2. start suite-destkop
3. reload

all looks ok using my setup. I can always see 2 devices. Could you please retest @bosomt and maybe provide more detailed repro steps?

**Comment by bosomt** [comment]
**QA OK**

im not able to reproduce issue anymore
/i was alble to replicate it few days ago/
if needed i will create separate issue 

**Info:**
 - Suite version: desktop 24.10.0 (3e1d4dff6ceb7a96cff59c4d15d223b0802366ca)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/24.10.0 Chrome/126.0.6478.234 Electron/31.6.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T2T1 2.8.1 regular (revision 632b9561559b7ab6824bb7eeac072874e07b7b82)
 - Transport: BridgeTransport 3.0.0-bundled.24.10.0
